.gitignore

```diff
@@ -17,4 +17,9 @@ static/stylesheets/no-mq.css
 static/stylesheets/style.css
 __pycache__
 *.db
+*.py[co]
```

Comment:
While I also think a global .gitignore is good, I don't see it as a good thing for the maintainer side. The world is wild out there. Providing a .gitignore with things we don't want to get into the codebase for me is the right way to go. Also, this facilitates open source contribution.